src/main/java/com/github/havardh/javaflow/phases/parser/java/JavaParser.java

```diff
@@ -67,16 +67,12 @@
 
   private static boolean containsEnum(CompilationUnit cu) {
     return cu.getTypes().stream()
-        .filter(t -> t instanceof EnumDeclaration)
-        .findFirst()
-        .isPresent();
+        .anyMatch(t -> t instanceof EnumDeclaration);
```

Comment:
This is just cosmetics (it's a bit shorter), but I can remove it again if you don't like it of course.